> what can I say :(
> 
> ```
>  -- Build files have been written to: /__w/zephyr/zephyr/twister-out/mps3_corstone310_fvp_ns/samples/psa/its/sample.psa.its.tfm/tfm/lib/ext/ethos_u_core_driver-subbuild
> [1/9] Creating directories for 'ethos_u_core_driver-populate'
> [1/9] Performing download step (git clone) for 'ethos_u_core_driver-populate'
> Cloning into 'ethos_u_core_driver-src'...
> fatal: unable to access 'https://review.mlplatform.org/ml/ethos-u/ethos-u-core-driver/': Failed to connect to review.mlplatform.org port 443 after 129420 ms: Connection timed out
> Cloning into 'ethos_u_core_driver-src'...
> fatal: unable to access 'https://review.mlplatform.org/ml/ethos-u/ethos-u-core-driver/': Failed to connect to review.mlplatform.org port 443 after 131045 ms: Connection timed out
> Cloning into 'ethos_u_core_driver-src'...
> fatal: unable to access 'https://review.mlplatform.org/ml/ethos-u/ethos-u-core-driver/': The requested URL returned error: 502
> -- Had to git clone more than once:
>           3 times.
> CMake Error at ethos_u_core_driver-subbuild/ethos_u_core_driver-populate-prefix/tmp/ethos_u_core_driver-populate-gitclone.cmake:31 (message):
>   Failed to clone repository:
>   'https://review.mlplatform.org/ml/ethos-u/ethos-u-core-driver'
> ```

I have the same issue. Something broken in the pipeline?

--- a/subsys/bluetooth/controller/ll_sw/ull_filter.c
+++ b/subsys/bluetooth/controller/ll_sw/ull_filter.c
@@ -1164,7 +1164,7 @@ static void rpa_adv_refresh(struct ll_adv_set *adv)
 	uint8_t pri_idx;
 
 #if defined(CONFIG_BT_CTLR_ADV_EXT)
-	uint8_t sec_idx;
+	uint8_t sec_idx = UINT8_MAX;
 #endif /* CONFIG_BT_CTLR_ADV_EXT */
 
 	if (adv->own_addr_type != BT_HCI_OWN_ADDR_RPA_OR_PUBLIC &&

